3-UsingAPIs/Week2/homework/ex4-diceRace.js

```diff
@@ -15,18 +16,24 @@ const rollDie = require('../../helpers/pokerDiceRoller');
 
 function rollDice() {
   const dice = [1, 2, 3, 4, 5];
-  // TODO complete this function; use Promise.race() and rollDie()
+  const raceDice = dice.map(rollDie);
```

Comment:
The name `raceDice` leaves me guessing what the variable could contain. Better be a bit more explicit if possible.

```suggestion
  const promises = dice.map(rollDie);
```

or perhaps:

```js
const rollDiePromises = dice.map(rollDie);
```